app/views/policy_classes/_form_controls.html.erb

```diff
@@ -30,10 +30,14 @@
     <% end %>
   </div>
 <% end %>
+
+<% if planning_application.submit_recommendation_complete? %>
+  <%= back_link %>
+<% else %>
 <%= render(
   partial: "shared/submit_buttons",
   locals: {
-    form: form,
-    disabled: planning_application.submit_recommendation_complete?
+    form: form
   }
 ) %>
+<% end %>
```

Comment:
Sorry if I've misunderstood - if submitted is it supposed to show an edit link? Not sure at the moment it will